sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 4.0.0-preview.9 (2019-11-04)
+
+- Fixed parameter names and access modifiers to be on par of other language implementations.
+- Updated dependencies.
+- Cleaned up the exported types.
+- Renamed `Certificate` to `KeyVaultCertificate`.
+- Ensured that all the optional parameters have this format `{PascalCaseMethodName}Options`.
+- Date properties now end in `On`.
+- Added braces `[]` around optional parameters.
```

Comment:
As mentioned once before changelog should reflect points that affect the user. The `[]` has no net affect on the user's experience with the library, so can be omitted.

```suggestion
```